MetaMask/metamask-extension
# Fix for hovering any Address parsed on Signed Type Data screen does not display the pointer anymore

## Explanation

Added fix, so now when hovering over address on Signed Type Data screen cursor pointer shows up. 
* Fixes #17860

## Screenshot
### Before

![signTypeDataBug](https://user-images.githubusercontent.com/120601632/223695901-442917b7-3cae-480d-a6fa-433dbbf62f7f.gif)

### After

![signTypeDataFix](https://user-images.githubusercontent.com/120601632/223695183-b1a52267-9b63-466d-84a3-df6b70ba1211.gif)

## Unit tests

![Screenshot 2023-03-07 at 11 43 58](https://user-images.githubusercontent.com/120601632/223409690-be969205-c6c9-4b70-8be4-de764b816b40.png)

## Lint

<img width="696" alt="Screenshot 2023-03-08 at 11 51 40" src="https://user-images.githubusercontent.com/120601632/223695122-e741ae82-b139-4e28-9d69-def1edc1424c.png">

## Manual Testing Steps

1. Go to Test Dapp https://metamask.github.io/test-dapp/
2. Connect MetaMask
3. Click to any Sign type data (Sign Type Data V4 is easiest to see)
4. Hover any address -- see mouse pointer is now shown
